### ISSUE TITLE
[Feature Request] Rescan for new files during upload

### ISSUE THREAD

**Issue by flixajki** [ncw/rclone#840]
It would be great if rclone could check every minute if there are any additional files for upload and add them to current task, assuming you have "free transfers" left.

As described in this topic: https://forum.rclone.org/t/rescan-during-upload-add-new-files-to-que/158

**Comment by marrie** [comment]
While I like the idea, would that not cause an infinite loop if new files are constantly being added to the directory and old ones are being changed? Just my thoughts there.

**Comment by flixajki** [comment]
Yup it would and it should be optional eg with switch.

**Comment by marrie** [comment]
Agreed.  maybe a flag that you can set the interval for scanning this way it takes up less system resources.

**Comment by ncw** [comment]
I think this would be hard to implement the way rclone is structured.  I'll leave it as an enhancement though just in case I or anyone else has an idea about how to implement it.

**Comment by marrie** [comment]
Thanks for that.  For now we'll just work with what we have, which is a wonderful tool for backing up data.

Thanks again.
